docs/user/azure/tested_instance_types_aarch64.md

```diff
@@ -0,0 +1,2 @@
+* `standardD4psv5`
+* `standardD8psv5`
```

Comment:
we can probably change this to `standardDpsv5` family similar to x86. we could add more arm types. let me see if i can come up with all the arm instances